pypesto/petab/importer.py

```diff
@@ -417,6 +424,31 @@ def create_objective(
             # FIXME: currently not supported with hierarchical
             kwargs['guess_steadystate'] = False
 
+        if self._ordinal:
+            # TODO add constants to C
+            if 'inner_problem_method' in kwargs:
+                inner_problem_method = kwargs.pop('inner_problem_method')
+            else:
+                inner_problem_method = None
+            if 'inner_solver_options' in kwargs:
+                inner_solver_options = kwargs.pop('inner_solver_options')
+            else:
+                inner_solver_options = None
```

Comment:
```suggestion
            inner_solver_options = kwargs.get('inner_solver_options', None)
```

Reply:
Think get will not work, as described up.
But I put this: 
            inner_problem_method = kwargs.pop(INNER_SOLVER_OPTIONS, None)

Reply:
:+1: